### DISCUSSION TITLE
Self-built MeshLab moving slow

### DISCUSSION THREAD

**Discussion by flaviu22** [cnr-isti-vclab/meshlab#864] [Q&A] [answered]
I have compiled MeshLab with Qt, as debug, on Win10 64bit. Successfully. 

![image](https://user-images.githubusercontent.com/37621568/102087681-93896f80-3e22-11eb-86d0-3c9b32fb85d1.png)

But I noticed soomething strange: the viewer is moving slow and eat a lot of CPU resources. Ok then, I have installed the meshlab app from here: https://www.meshlab.net/#download

And this one is moving fast. Why ? I have searched on app settings, nothing related found. Is this a problem that I have compiled as Debug ? This could make so much difference ? What I should do with my Qt generated app in order to move it well ?

Moreover, when I close the app I got:

![image](https://user-images.githubusercontent.com/37621568/102087805-bb78d300-3e22-11eb-9c61-d6eb7932b512.png)

This is related problem ? Should I open another issue for this issue ?

**Comment by alemuntoni** [comment, marked as answer]
I am pretty sure that is moving slow because you are in debug mode (mostly if your PC is not very powerful). 
Moreover, on my experience, being on Windows and building with MSVC does not help. Try to build on Release mode.

About the second issue, never saw something like that.

--
I moved this thread to discussions since these kind of issues are not strictly related to the MeshLab versions we release.
